instagram/src/App.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import dummyData from './Dummy';
+import SearchBar from './components/SearchBar/searchbar';
+import PostContainer from './components/PostContainer/postcontainer';
+import PropTypes from 'prop-types';
+
+class App extends Component {
+  render() {
+    return (
+      <div className ="app-container">
+        <SearchBar />
+        <div>
+          {dummyData.map(dummyObj =>{
+            return(
+              <PostContainer dummyObj = {dummyObj} key = {dummyObj.timestamp}/>
+            );
+          })}
+        </div>
+      </div>
+    );
+  }
+}
+App.propTypes = {
```

Comment:
App doesn't receive props, from what I can tell, so this doesn't make sense here